<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepResult</class>
 <widget class="QDialog" name="StepResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select result step/ Animation</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>16</width>
     <height>202</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Animation Control</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Start Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="startCombo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>End Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="endCombo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Delay (mili seconds)        </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="delayLine"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="autoChangeValueCheckBox">
      <property name="text">
       <string>Auto Change Max Min Values</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="animationStart">
      <property name="text">
       <string>Start Animation</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>331</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Single Step Control</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Results field</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="resultCombo">
        <item>
         <property name="text">
          <string>X-Displacement</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Y-Displacement </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Z-Displacement</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>EPWP- Pore pressure</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Material Type</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Element Type</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="stepCombo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Max and Min values</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="maxCombo">
        <item>
         <property name="text">
          <string>Auto Calculate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manually</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Max Value                                      </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="maxLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Min Value                                       </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="minLine"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>Ok/Replot</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
